index.html

```diff
@@ -1391,6 +1402,16 @@
           used to probe information about which <a>remote playback device</a>
           the user has without user consent.
         </p>
+        <p>
+          If a user <a data-lt="disable remote playback">disables remote
+          playback</a> on all <a data-lt="media element">media elements</a>
+          (through a browser setting, for example), the user agent should
+          stop <a data-lt="monitor the list of available remote playback devices">
+          monitoring the list of available remote playback devices</a>
+          and will clear the <a>global set of availability callbacks</a>. In
+          this state, pages will receive no further information about the
```

Comment:
I think this is an issue with the steps to disable remote playback.  It clears the callbacks and does not invoke them with `false` before doing so.